spec/factories/cards.rb

```diff
@@ -2,6 +2,7 @@
   factory :card do
     original_text   { Faker::Lorem.word }
     translated_text { Faker::Name.name }
+    image_card      File.open("default_img.jpg")
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.